lib/datasource/common.ts

```diff
@@ -26,6 +26,8 @@ export interface Release {
 export interface ReleaseResult {
   sourceDirectory?: string;
   latestVersion?: string;
+  license?: string;
+  licenseUrl?: string;
```

Comment:
Does this point to the license (e.g. Apache 2.0 definition) or points to where you found the license?

Reply:
This points to the actual license file the repo is using, usually a github.com link